app/controllers/goals_controller.rb

```diff
@@ -12,18 +12,25 @@ def show
       scores =
         scores.reduce({}) do |acc, score|
           key = "#{assessment_type}_ind_#{score['indicator_id']}"
-          acc[key] = score['score']
+          acc[key] = Score.new score['score']
           acc
         end
 
+      if assessment_type == 'spar_2018'
```

Comment:
will this key change on a yearly basis? why not make the assessment type just `spar`?

Reply:
Hedging against spar_2020 or some such. Maybe premature?